cfgov/jinja2/v1/_includes/organisms/info-unit-group-2.html

```diff
@@ -58,20 +64,36 @@
 {%- if value.intro %}
     {{ value.intro | safe }}
 {%- endif %}
-
-    <div class="content-l">
+  {% if not value.lines_between_items %}
+  <div class="content-l">
+  {% endif %}
     {% for unit in value.info_units %}
+        {% if value.lines_between_items and  (
+          ( loop.index == 1 ) or
```

Comment:
Minor, Indentation should be four spaces.

Reply:
Should be fixed in https://github.com/cfpb/cfgov-refresh/pull/3476/commits/cd56484fa764a5930cc8390511c7c89ca8bb1889.